common/persistence/persistenceErrorInjectionClients.go

```diff
@@ -2354,13 +2354,13 @@ func (p *configStoreErrorInjectionPersistenceClient) FetchDynamicConfig(ctx cont
 	return response, persistenceErr
 }
 
-func (p *configStoreErrorInjectionPersistenceClient) UpdateDynamicConfig(ctx context.Context, request *UpdateDynamicConfigRequest) error {
+func (p *configStoreErrorInjectionPersistenceClient) UpdateDynamicConfig(ctx context.Context, request *UpdateDynamicConfigRequest, cfgType ConfigType) error {
 	fakeErr := generateFakeError(p.errorRate)
 
 	var persistenceErr error
 	var forwardCall bool
 	if forwardCall = shouldForwardCallToPersistence(fakeErr); forwardCall {
-		persistenceErr = p.persistence.UpdateDynamicConfig(ctx, request)
+		persistenceErr = p.persistence.UpdateDynamicConfig(ctx, request, DynamicConfig)
```

Comment:
should be `cfgType`

Reply:
? DynamicConfig is a type of value ConfigType
<img width="565" alt="Screenshot 2023-03-29 at 5 35 49 PM" src="https://user-images.githubusercontent.com/2725764/228698253-20f9e670-b02e-4e87-a422-0e9658f8f0a6.png">

Reply:
you should copy the param

Reply:
yes 🤦  thank you